src/util/event-emitter.js

```diff
@@ -21,14 +19,14 @@ export class EventEmitter{
   }
 
   next(value){
-    this._subject.onNext(value);
+    this._subject.next(value);
   }
 
   throw(error){
-    this._subject.onError(error);
+    this._subject.error(error);
   }
 
   return(){
-    this._subject.onCompleted();
+    this._subject.complete();
   }
 }
```

Comment:
Click on the "command line instructions" link below. It will give you pretty clear instructions.